usecases/backup/backend.go

```diff
@@ -124,7 +126,8 @@ Loop:
 
 // class uploads one class
 func (u *uploader) class(ctx context.Context, id string, desc backup.ClassDescriptor) (err error) {
-	// TODO and instrumentation
+	timer := prometheus.NewTimer(monitoring.GetMetrics().BackupStoreDurations.WithLabelValues(getType(u.backend.BackupBackend), desc.Name))
```

Comment:
WithLabelValues might panic and we don't want to panic because of a metric.
You could use GetMetricWithLabelValues instead 

This applies to the whole PR